Makefile

```diff
@@ -144,15 +146,7 @@ instrument-binary:
 
 coverage:
 	rm -f coverage.txt
-	echo 'mode: count' > coverage.txt
-	set -e ; for p in $(PKGS); do \
-		rm -f coverage-tmp.txt;  \
-		$(GO) test -covermode=count -coverprofile=coverage-tmp.txt -coverpkg=$(SUBPKGS) $$p ; \
-		if [ -f coverage-tmp.txt ]; then \
-			cat coverage-tmp.txt |grep -v 'mode:' | cat >> coverage.txt; \
-		fi; \
-	done
-	rm -f coverage-tmp.txt
+	go test -tags '$(TAGS)' -covermode=atomic -coverprofile=coverage.txt ./...
```

Comment:
I think that this is the reason for the coverage drop. I don't remember the details, but I looked in this years ago and ended up concluding that it is necessary to do package by package :man_shrugging: